dbt/include/athena/macros/materializations/seeds/helpers.sql

```diff
@@ -14,18 +14,27 @@
   {%- set s3_data_naming = config.get('s3_data_naming', target.s3_data_naming) -%}
   {%- set external_location = config.get('external_location', default=none) -%}
 
+  {%- set s3_location = adapter.upload_seed_to_s3(
+    s3_data_dir,
+    s3_data_naming,
+    external_location,
+    model.schema,
+    model.name,
+    agate_table,
+  ) -%}
+
   {% set sql %}
-    create external table {{ this.render_hive() }} (
+    CREATE EXTERNAL TABLE {{ this.render_hive() }} (
         {%- for col_name in agate_table.column_names -%}
             {%- set inferred_type = adapter.convert_type(agate_table, loop.index0) -%}
             {%- set type = column_override.get(col_name, inferred_type) -%}
             {%- set column_name = (col_name | string) -%}
             {{ adapter.quote_seed_column(column_name, quote_seed_column) }} {{ type }} {%- if not loop.last -%}, {% endif -%}
         {%- endfor -%}
     )
-    stored as parquet
-    location '{{ adapter.s3_table_location(s3_data_dir, s3_data_naming, model["schema"], model["alias"], external_location) }}'
-    tblproperties ('classification'='parquet')
+    ROW FORMAT SERDE 'org.apache.hive.hcatalog.data.JsonSerDe'
+    WITH SERDEPROPERTIES ("timestamp.formats"="yyyy-MM-dd'T'HH:mm:ss.SSS'Z',yyyy-MM-dd'T'HH:mm:ss,yyyy-MM-dd HH:mm:ss")
+    LOCATION '{{ s3_location }}'
```

Comment:
Maybe here you could add a todo and consider the workgroup location if set up.

Reply:
👍 I'll add a comment on `upload_seed_to_s3` in the adapter code